package/cloudgate/data/etc/init.d/helium_gateway

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2006 OpenWrt.org
+STOP=80
+
+start() {
+	/rom/mnt/cust/usr/bin/monit -g helium restart all
+}
+
+stop() {
+	/rom/mnt/cust/usr/bin/monit -g helium stop all
+}
+
+restart() {
+	/rom/mnt/cust/usr/bin/monit restart helium_cg
```

Comment:
why is this called `helium_cg`?

Reply:
helium_cg is a process that waits until there is an internet connection and starts the gateway_rs.
restarting, in this case, sends a signal to this process "helium_cg" that will in his turn stop the gateway_rs and then restarts it (when there is an internet connection of course).